guides/de-fr-xinvoice.mdx

```diff
@@ -0,0 +1,611 @@
+            }
+        ],
+        "receiving": [
+            {
+            "code": "3544"
+            }
+        ],
+        "despatch": [
+            {
+            "code": "5433"
+            }
+        ]
+    },
+    "payment": {
+        "payee": {
+            "name": "Ebeneser Scrooge AS",
+            "identities": [
+            {
+                "label": "CompanyID",
+                "code": "989823401"
+            },
+            {
+                "label": "0088",
+                "code": "2298740918237"
+            }
+            ]
+        },
+        "terms": {
+            "due_dates": [
+            {
+                "date": "2013-07-20",
+                "amount": "2142.00",
+                "percent": "100%"
+            }
+            ],
+            "notes": "2 % discount if paid within 2 days\n            Penalty percentage 10% from due date"
+        },
+        "instructions": {
+            "key": "credit-transfer",
+            "ref": "0003434323213231",
+            "ext": {
+                "untdid-payment-means": "30"
+            },
+            "credit_transfer": [
+            {
+                "iban": "NO9386011117947",
+                "bic": "DNBANOKK"
+            }
+            ],
+            "direct_debit": {
+                "creditor": "987654321",
+                "account": "DE89370400440532013000",
+                "ref": "1234567890"
+            },
+            "card": {
+                "last4": "1234",
+                "holder": "John Doe"
+            }
+        }
+    },
+    "delivery": {
+        "receiver": {
+            "addresses": [
+                {
+                    "street": "Deliverystreet 2",
+                    "street_extra": "Side door",
+                    "locality": "DeliveryCity",
+                    "region": "RegionD",
+                    "code": "523427",
+                    "country": "NO"
+                }
+            ]
+        },
+        "identities": [
+            {
+                "label": "0088",
+                "code": "6754238987643"
+            }
+        ],
+        "date": "2024-02-10"
+    },
+    "tax": {
+        "ext": {
+            "untdid-document-type": "380"
+        }
+    },
+    "totals": {
+        "sum": "1800.00",
+        "total": "1800.00",
+        "taxes": {
+            "categories": [
+                {
+                    "code": "VAT",
+                    "rates": [
+                        {
+                            "key": "standard",
+                            "base": "1800.00",
+                            "percent": "19%",
+                            "amount": "342.00"
+                        }
+                    ],
+                    "amount": "342.00"
+                }
+            ],
+            "sum": "342.00"
+        },
+        "tax": "342.00",
+        "total_with_tax": "2142.00",
+        "payable": "2142.00"
+    }
+}
+```
+</Accordion>
+</AccordionGroup>
+
+## Example Workflow
```

Comment:
I'd suggest moving the example workflow up to the setup section like in TBAI or Mexico.